src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs

```diff
@@ -63,10 +63,16 @@ public CameraDeviceManager()
         /// Gets the status whether camera device(usb, network) is connected or not.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public bool IsExternalCameraConnected =>
-            GetDeviceInformation().Where(d => d.Type == CameraDeviceType.Usb ||
-                                              d.Type == CameraDeviceType.Network)
-                                  .Any();
+        public bool IsExternalCameraConnected
+        {
+            get
+            {
+                var deviceList = GetDeviceInformation();
+
+                return deviceList == null ? false : deviceList.Where(d => d.Type == CameraDeviceType.Usb ||
+                                                                          d.Type == CameraDeviceType.Network).Any();
+            }
+        }
```

Comment:
How about use `?.` null conditon operator?
```suggestion
        public bool IsExternalCameraConnected =>
            GetDeviceInformation()?.Where(d => d.Type == CameraDeviceType.Usb ||
                                              d.Type == CameraDeviceType.Network)
                                  .Any() ?? false;
```

Reply:
I changed the return type of GetDeviceInformation(), null to Empty.
So, I think we don't need to check anymore whether it's null or not. I'll change this like the following.
```
public bool IsExternalCameraConnected =>
    GetDeviceInformation().Where(d => d.Type == CameraDeviceType.Usb ||
                                      d.Type == CameraDeviceType.Network)
                          .Any();
```